src/diffpy/structure/atom.py

```diff
@@ -261,12 +398,49 @@ def _set_U(self, value):
     # Uij elements
 
     def _get_Uij(self, i, j):
+        """
+        Obtain the elements of anisotropic thermal displacement tensor ``Uij``.
+
+        Parameters
+        ----------
+        i : int
+            The index of anisotropic thermal displacement tensor ``Uij``.
+            i is from (1, 2, 3).
+        j : int
+            The index of anisotropic thermal displacement tensor ``Uij``.
+            j is from (1, 2, 3).
+
+        Returns
+        -------
+        float
+            The elements of anisotropic thermal displacement tensor ``Uij``
+        """
+
         if self.anisotropy:
             return self._U[i, j]
         lat = self.lattice or cartesian_lattice
         return self._U[0, 0] * lat.isotropicunit[i, j]
 
     def _set_Uij(self, i, j, value):
+        """
+        Set the elements of anisotropic thermal displacement tensor ``Uij``.
+
+        Parameters
+        ----------
+        i : int
+            The index of anisotropic thermal displacement tensor ``Uij``.
+            i is from (1, 2, 3).
+        j : int
+            The index of anisotropic thermal displacement tensor ``Uij``.
+            j is from (1, 2, 3).
+        value: float
+            The element of anisotropic thermal displacement tensor to be set.
+
+        Returns
+        -------
+        None.
+        """
+
```

Comment:
Remove docstrings for `_set_Uij`, `_get_Uij` functions.  Instead update the `doc` argument in property definitions of `U11`, `U22`, etc.